homeassistant/components/homekit/__init__.py

```diff
@@ -44,6 +44,8 @@
     DEVICE_CLASS_CO2,
     DEVICE_CLASS_PM25,
     DOMAIN,
+    EVENT_HOMEKIT_STARTED,
```

Comment:
This is the wrong implementation. Please use the `logbook.async_log_entry` helper method.

Reply:
ok, ive made adjustments here to use the helper.